src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -1,5 +1,6 @@
 (ns org.broadinstitute.firecloud-ui.page.workspace.summary.attribute-editor
   (:require
+    clojure.set
```

Comment:
Yeah it's used in the file. I wish compilation was more strict about dependencies.